scheduler/scheduler.go

```diff
@@ -119,6 +126,11 @@ type Scheduler struct {
 	invocationID string
 }
 
+type shard struct {
+	num   int32
```

Comment:
Why `int32`?

Reply:
For context https://github.com/cloudquery/plugin-pb/pull/26#pullrequestreview-2310265498, `int32` comes from protobuf type support (no `int`)